instance/server/Client/Admin/Views.elm

```diff
@@ -11,12 +11,12 @@ import Client.Components exposing (..)
 import Client.Styles exposing (..)
 import Shared.Test exposing (..)
 import Shared.User exposing (..)
-import Shared.Routes exposing (..)
+import Shared.Routes exposing (Route(..), toPath, assets)
 
 
 loginView =
     form
-        [ action routes.login
+        [ action (toPath Login)
```

Comment:
What I personally do not like about helpers is necessity to know about them and you pollute developer mind with this knowledge.

You can check [this talk](https://www.youtube.com/watch?v=4anAwXYqLG8) where it's being described.

I think better would be to refactor it to `action <| toPath login`. If you anyway think helper would be better I'll add it.